### PR TITLE
[9.0][MIG][product_custom_info] Migrated to v9

### PR DESCRIPTION
**Pull request by yajo**
Allow to have custom information attached to products, sorted by templates.

WIP until:
- [x] https://github.com/OCA/server-tools/pull/492

... Because it inherits its changes.

@Tecnativa

### CODE REVIEW

File: product_custom_info/views/product_view.xml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                  string="Custom Information">
+                <group>
+                    <field
+                        name="custom_info_template_id"
+                        options='{"no_quick_create": True}'/>
+                    <field
+                        name="custom_info_ids"
+                        attrs="{'invisible': [('custom_info_template_id', '=', False)]}">
+                        <tree create="0" delete="0">
+                            <field name="category_name"/>
```
Comment (sergio-teruel) [inline product_custom_info/views/product_view.xml]: When I try to install the module this field do not exists..

File: product_custom_info/views/product_view.xml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+                            <field name="property_id"/>
+                            <field name="value"/>
+                        </tree>
+                        <form>
+                            <field name="field_type" invisible="True"/>
+                            <field name="category_name"/>
```
Comment (sergio-teruel) [inline product_custom_info/views/product_view.xml]: The same

**Comment by guewen** [comment]
(no test, code review) :+1:

**Comment by rafaelbn** [comment]
@guewen could you please revew https://github.com/OCA/server-tools/pull/492 first? We must merge that one before this one. Thanks!

**Comment by pedrobaeza** [comment]
@rafaelbn that PR is not yet ready to be reviewed.

**Comment by yajo** [comment]
OCA/server-tools#492 was merged, so I squashed and rebased, let's see if all goes 💚 now.

Please somebody tag as ready to review

**Review by HaTry — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**